drivers/counter/counter_ll_stm32_rtc.c

```diff
@@ -106,7 +107,7 @@ static uint32_t rtc_stm32_read(const struct device *dev)
 
 	/* Convert calendar datetime to UNIX timestamp */
 	/* RTC start time: 1st, Jan, 2000 */
-	/* time_t start:   1st, Jan, 1900 */
+	/* time_t start:   1st, Jan, 1970 */
 	now.tm_year = 100 +
 			__LL_RTC_CONVERT_BCD2BIN(__LL_RTC_GET_YEAR(rtc_date));
```

Comment:
I agree the comment update is correct, but this whole section (comment + code) is now getting tricky.
It is likely that time conversion philistine will conclude to a wrong comment.
That said, I'm not quite sure how to set that in a clearer way.